app/pages/multisig/updates/CreatePLT/CreatePltParameters.tsx

```diff
@@ -0,0 +1,216 @@
+                        return true;
+                    },
+                }}
+            />
+            <Form.Input
+                className="body2 mB20"
+                name={fieldNames.name}
+                label={`${fieldDisplays.name}`}
+                placeholder="Token Name"
+                rules={{ required: requiredMessage(fieldDisplays.name) }}
+            />
+            <Form.Input
+                className="body2 mB20"
+                name={fieldNames.moduleRef}
+                label={`${fieldDisplays.moduleRef}`}
+                defaultValue="5c5c2645db84a7026d78f2501740f60a8ccb8fae5c166dc2428077fd9a699a4a"
+                rules={{
+                    required: requiredMessage(fieldDisplays.moduleRef),
+                    validate: (value: string) => {
+                        if (
+                            value !==
+                            '5c5c2645db84a7026d78f2501740f60a8ccb8fae5c166dc2428077fd9a699a4a'
+                        ) {
+                            return 'Protocol 9 only supports token module hash `5c5c2645db84a7026d78f2501740f60a8ccb8fae5c166dc2428077fd9a699a4a`.';
+                        }
+                        return true;
+                    },
+                }}
+            />
+            <Form.Input
+                className="body2 mB20"
+                name={fieldNames.metadataUrl}
+                label={`${fieldDisplays.metadataUrl}`}
+                placeholder="https://tokenWebsite.com"
+                rules={{ required: requiredMessage(fieldDisplays.metadataUrl) }}
+            />
+            <Form.Input
+                className="body2 mB20"
+                name={fieldNames.metadataHash}
+                label={`${fieldDisplays.metadataHash} (Optional)`}
+                placeholder="b27a46456f5d3c089f7ad76bbbc3525ef277532b131ffadfc2565094c4b5133a"
+                rules={{
+                    validate: (hash?: string) => {
+                        if (!hash) {
+                            // Allow undefined (no checksum hash)
+                            return true;
+                        }
+                        // Check if hash is set that it is a valid hash.
+                        if (!/^[a-f0-9]{64}$/.test(hash)) {
+                            return 'Hash must be a 64-character hexadecimal string';
+                        }
+                        return true;
+                    },
+                }}
+            />
+            <Form.Input
+                className="body2 mB20"
+                name={fieldNames.governanceAccount}
+                label={`${fieldDisplays.governanceAccount}`}
+                placeholder="4BTFaHx8CioLi8Xe7YiimpAK1oQMkbx5Wj6B8N7d7NXgmLvEZs"
+                rules={{
+                    required: requiredMessage(fieldDisplays.governanceAccount),
+                    validate: (governanceAccount: string) => {
+                        // TODO: check that governance account exists on-chain
```

Comment:
Can we eliminate this todo? async validation is supported, so it should not be an issue to check.

Reply:
Once we finalize the exact validation to do in `concordium-client`, the suggestion would be to do the same validation in the desktop wallet. Seems that likely we would only need to add the check if the governance account exists on-chain to DW hence that commnet. 

https://linear.app/concordium/issue/COR-1645/concordium-client-validation-improvements-add-better-error-reporting